volttrontesting/fixtures/environment_fixtures.py

```diff
@@ -0,0 +1,117 @@
+import contextlib
+import os
+import shutil
+from types import SimpleNamespace
+
+from volttron.platform import update_platform_config
+from volttron.utils import get_random_key
+from volttrontesting.fixtures.cert_fixtures import certs_profile_1
+from volttrontesting.utils.platformwrapper import create_volttron_home
+from volttrontesting.utils.utils import get_hostname_and_random_port, get_rand_ip_and_port
+
+
+@contextlib.contextmanager
+def build_test_environment(messagebus: str, web_https=False, web_http=False, has_vip=True,
```

Comment:
Doesn't seem to be used anywhere. Where is this used?
There is another get_test_volttron_home() in web_utils which is used. Both seems similar. Can it be combined and moved to volttron_platform_fixtures?

Reply:
This function is going to be used to build all test environments...there should not need to be any other function that both creates and sets up the volttron_home and config file for volttron home (note haven't added rabbit stuff).  It's used in some of the tests that I have on my hard drive :)...  The goal with this is to make the volttron_home things and then clean up at the end...and have it in one place that we can use not only within the platform wrapper but also from random tests.  The one in get_test_volttron_home is being replaced by this one.